content/posts/blog/2023-07-07-summary-of-upcoming-changes-in-oci-image-and-distribution-specs-v-1-1.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Summary of Upcoming Changes in OCI Image and Distribution Specs v1.1
+author:
+  name: Open Container Initiative
+  tag: oci
+tags: blog
+date: 2023-07-07
+---
+
+As mentioned in an [earlier post](./2023-01-26-oci-in-2023.md), the
+OCI Reference Types Working Group delivered a suggested set of changes
+in the OCI Image and Distribution specs to enable the creation and
+discovery of relationships between objects stored in an OCI registry.
+
+These changes (with some subsequent modifications) will be included in
+upcoming minor releases of both OCI Image and Distribution spec.
+
+# Notes on Versioning and the Term "OCI 1.1"
+
+The version of the upcoming minor releases of both OCI Image and
+Distribution spec happens to be the same: `v1.1.0`.
```

Comment:
It seems opencontainers-specs are self-aligning, as the runtime spec is also looking to release [1.1.0](https://github.com/opencontainers/runtime-spec/pull/1213) any moment now...

Reply:
yes, but not intentional 😅 may not always be the case in the future